### DISCUSSION TITLE
Internal Server Error after reboot of Controller

### DISCUSSION THREAD

**Discussion by bmcgonag** [openwisp/ansible-openwisp2#492] [Q&A] [answered]
I've done this a few times now. I am putting the controller on a ubuntu 22.04 LXD container (Incus, but same thing), and am allowing nesting and privileged.  

After running the ansible playbook it comes up fine, and I can change the admin password, and even register a device. 

When I reboot this server though, then i only get 'Internal Server Error' in the web ui when I go back to the page.  

Any idea what this could be?

**Comment by nemesifier** [comment, marked as answer]
Look for errors in the logs, find them at /opt/openwisp2/log/.
